### PR TITLE
[CDAP-11656] Add navigation link to Data Preparation workspaces and connections

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-11656

@bdmogal I was not sure from the JIRA whether we want to add the `BETA` tag to the navbar as well or is it for the Data Prep title inside the page?

### CODE REVIEW

File: cdap-ui/app/cdap/components/Header/ProductDropdown/ProductDropdown.scss [hunk 1]
```diff
@@ -192,15 +192,15 @@ $cdap_mode_color: #cccccc;
...
+  // .beta-badge {
+  //   font-size: 9px;
+  //   background-color: #5cb85c;
+  //   border-radius: 4px;
+  //   color: white;
+  //   margin-left: 2px;
+  //   padding: 2px 3px;
+  //   position: relative;
+  //   top: -7px;
+  // }
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/Header/ProductDropdown/ProductDropdown.scss]: Will be removed or location moved depending on whether we want to add beta tag to the navbar or not.

**Comment by bdmogal** [comment]
@elfenheart yes, I think we should have a BETA tag in the navbar link.

**Comment by bdmogal** [comment]
There is a thought that Workspaces is a confusing term, and that we don't use it anywhere other than the main navbar. Since it is so late in the release, instead of introducing a new flow, let's remove the dropdown on Data Preparation, and have one link on the navbar that behaves like the current Workspaces link - goes to the data prep table page if there are tabs, and to connections when there are no tabs.

Also, please use the new specs for the Beta label.

**Review by bdmogal — APPROVED** [review]

**Comment by tonybach** [comment]
Code LGTM :+1:

**Pull request merged**